Converting Aspect to JSON-LD (#651)
eclipse-esmf/esmf-sdk: documentation/developer-guide/modules/tooling-guide/pages/maven-plugin.adoc

| `outputDirectory` | The path to the directory where the generated JSON payload will be written to. | `String` | none | {ok}
|===

> === Generating Sample JSON-LD

Reviewer: This does not create sample data for an Aspect Model, but a JSON-LD representation of the model itself.
```suggestion
=== Generating JSON-LD representation of an Aspect Model
```

> The `generateJsonLd` goal generates a valid sample JSON-LD for a given Aspect Model as it
> could be returned by an Aspect that implements that Aspect Model. The default life cycle phase for
> the goal is `generate-resources`.

Reviewer: ```suggestion
The `generateJsonLd` goal generates a JSON-LD represention of a given Aspect Model. The default life cycle phase for the goal is `generate-resources`.
```
